v2/dtos/propertyvalue.go

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+// PropertyValue defines constraints with regard to the range of acceptable values assigned to an event reading and defined as a property within a device profile.
+// This object and its properties correspond to the PropertyValue object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/PropertyValue
+type PropertyValue struct {
```

Comment:
I think we can have more robust validation on all allowed values in the string properties, but OK with creating an issue to address later so we can move forward with the V2 implementation. The issue should be **More robust validation for all V2 DTO's**

Reply:
Agree to create a new issue to add more validation tags for V2 DTOs.